viam-cartographer_internal_test.go

```diff
@@ -243,6 +244,75 @@ func TestGetPositionEndpoint(t *testing.T) {
 	})
 }
 
+//nolint:dupl
```

Comment:
is this still necessary if we remove 257?

Reply:
With 257 removed, I still get: ` 284-310 lines are duplicate of `viam-cartographer_internal_test.go:256-282``

Reply:
can you please refactor it to not hit this lint rule by making a helper function?